compiler/passes/src/destructuring/ast.rs

```diff
@@ -272,7 +281,7 @@ impl AstReconstructor for DestructuringVisitor<'_> {
                 .collect()
         };
 
-        let (value, mut statements) = self.reconstruct_expression(definition.value);
+        let (value, mut statements) = self.reconstruct_expression(definition.value, &());
```

Comment:
Noticing a fair bit of `&()`, for additional input.
Just curious, is there a more cleaner way to specify it visually?

Reply:
All good, it just took me a bit of staring to get over it :)